[2025-03-04T05:11:22.817+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-04T05:11:22.837+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.thrid_task manual__2025-03-04T05:11:20.972033+00:00 [queued]>
[2025-03-04T05:11:22.845+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.thrid_task manual__2025-03-04T05:11:20.972033+00:00 [queued]>
[2025-03-04T05:11:22.845+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-04T05:11:22.854+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): thrid_task> on 2025-03-04 05:11:20.972033+00:00
[2025-03-04T05:11:22.862+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'thrid_task', 'manual__2025-03-04T05:11:20.972033+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpgg354azf']
[2025-03-04T05:11:22.864+0000] {standard_task_runner.py:105} INFO - Job 27: Subtask thrid_task
[2025-03-04T05:11:22.864+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=302) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-04T05:11:22.865+0000] {standard_task_runner.py:72} INFO - Started process 304 to run task
[2025-03-04T05:11:22.924+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v1.thrid_task manual__2025-03-04T05:11:20.972033+00:00 [running]> on host e52f68431fc8
[2025-03-04T05:11:23.008+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='thrid_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-04T05:11:20.972033+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-04T05:11:20.972033+00:00'
[2025-03-04T05:11:23.008+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-04T05:11:23.021+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-04T05:11:23.021+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task3 and will be running after task1 at the same time as task2!']
[2025-03-04T05:11:23.031+0000] {subprocess.py:99} INFO - Output:
[2025-03-04T05:11:23.032+0000] {subprocess.py:106} INFO - hey, I am task3 and will be running after task1 at the same time as task2!
[2025-03-04T05:11:23.033+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-04T05:11:23.058+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-04T05:11:23.059+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=thrid_task, run_id=manual__2025-03-04T05:11:20.972033+00:00, execution_date=20250304T051120, start_date=20250304T051122, end_date=20250304T051123
[2025-03-04T05:11:23.119+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-04T05:11:23.149+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-04T05:11:23.153+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
